src/shared/types/thoas/product.ts

```diff
@@ -43,6 +43,21 @@ export enum ProductType {
   PROTEIN = 'Protein'
 }
 
+type Location = {
+  start: number;
+  end: number;
+};
+
+type SequenceFamily = {
+  name: string;
+  description: string;
+  source: Source;
+};
+export type FamilyMatches = {
```

Comment:
Not matches, but match. This type is describing a single match.

Reply:
Done but just to make sure i got this right, further down we have the following
`family_matches: FamilyMatch[]`

does that makes sense? does that mean its an array of single family match?

Reply:
An array of  `FamilyMatch` makes sense. If not it should be a list of `FamilyMatches` below (e.g. `family_matches_list` instead of `family_matches`)?

Reply:
As Imran says, if we mean a list of family matches — which we do — then `FamilyMatch[]` makes perfect sense. Writing `FamilyMatch[]` is the same as writing `Array<FamilyMatch>`, which is to say an array of things, each of which is a family match. In contrast, a `FamilyMatches` type would suggest to the reader that it itself represents an array; and `FamilyMatches[]` would suggest that we are talking about an array of arrays of family matches.